sys/include/net/gnrc/netif/internal.h

```diff
@@ -300,32 +306,45 @@ int gnrc_netif_ipv6_get_iid(gnrc_netif_t *netif, eui64_t *eui64);
  * @return  true, if the interface represents a router
  * @return  false, if the interface does not represent a router
  */
+#if defined(MODULE_IPV6_ROUTER) || defined(DOXYGEN)
```

Comment:
Welp… that module doesn't exist (it's supposed to be `MODULE_GNRC_IPV6_ROUTER`)... Will fix.